.github/workflows/test-integrations.yml

```diff
@@ -29,25 +29,14 @@ env:
   # strip the hashicorp/ off the front of github.repository for consul
```

Comment:
No updates needed to triggers as they were already split on PR and protected branch push.